protocol/contracts/beanstalk/silo/ConvertFacet.sol

```diff
@@ -186,7 +193,8 @@ contract ConvertFacet is ReentrancyGuard {
                 token,
                 stems[i],
                 amounts[i],
-                bdv
+                bdv,
+                true // converting "burns" the old ERC1155 deposit, and "mints" a new ERC1155 deposit.
```

Comment:
Talk with @pizzaman1337 to see if this boolean would be better as an enum. I don't think it matters from a gas perspective (correct me if that's wrong) but would be way clearer from a code perspective.